index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="main-container">
+            <div class="header-container">
+                <div class="my-logo">
+                  <h1 class="logo">My Logo</h1>
```

Comment:
- [ ] [REQUIRED] Your logo should be an anchor tag so that it can be clickable.

Reply:
Thanks for the feedback I'll implement the required changes for the project